Soluciona l2q01.py #29

iamlalith opened the pull request:
solução para l2q03.py com comentários

iamlalith committed:
Soluciona l2q01.py #29

solução para l2q03.py com comentários

matheusvictor reviewed (changes requested):
Atente-se que a questão pede para não considerar valores iguais. Ou seja, é preciso garantir que o usuário não irá digitar duas vezes o número 3, por exemplo.

iamlalith committed:
solução atualizada I2qo1

Foi adicionada uma condição para informar ao usuário que não é necessário inserir números semelhantes

iamlalith commented:
alterações solicitadas feitas

matheusvictor commented:
@iamlalith : é preciso haver um laço de repetição que "force" o usuário a digitar novamente um número caso aquele já tenha sido digitado anteriormente. 

Com o código atual, é até possível reconhecer que algum número foi digitado repetido, mas é preciso executar o código novamente assim que o terceiro número é digitado.

Tente um looping ;D

matheusvictor reviewed (changes requested):
@iamlalith : é preciso haver um laço de repetição que "force" o usuário a digitar novamente um número caso aquele já tenha sido digitado anteriormente. 

Com o código atual, é até possível reconhecer que algum número foi digitado repetido, mas é preciso executar o código novamente assim que o terceiro número é digitado.

Tente um looping ;D

iamlalith committed:
solução atualizada I2q01

Adicionado recurso de repetição quando números digitados são semelhantes

iamlalith commented:
@matheusvictor adicionou um loop para que o usuário seja solicitado a inserir números novamente se os números já inseridos forem semelhantes

matheusvictor reviewed (approved):
@iamlalith: Ok! Está melhor atende ao enunciado da questão. Mas, uma sugestão: acredito que um loop indicando qual número digitado foi repetido e solicitar que outro número fosse solicitado no lugar deste, seria melhor. O que acha?! Fica aí a sugestão ;D
